apps/src/gamelab/GameLab.js

```diff
@@ -358,112 +398,7 @@ GameLab.prototype.execute = function() {
     }, this), 'divGameLab');
   /* jshint nonew:true */
 
-  if (this.level.editCode) {
```

Comment:
All of this init code has been moved into an `initInterpreter` method, which is called inside our `preload` function. If we left the code to run after the `new window.p5` was complete, the preload phase would already be over.